lib/money/rates_store/store_with_historical_data_support.rb

```diff
@@ -0,0 +1,6 @@
+require 'money/rates_store/eu_central_bank_historical_data_support'
+module Money::RatesStore
+  class StoreWithHistoricalDataSupport < Money::RatesStore::Memory
+    include Money::RatesStore::EuCentralBankHistoricalDataSupport
```

Comment:
Since this module is now only included in `StoreWithHistoricalDataSupport` (which doesn't have anything else), would it make more sense to merge the two and get rid of the module?

Reply:
You're right. It's unnecessary. Have moved it inside the class.